File: src/main/java/fr/lewon/web/bot/entities/response/ContinueQuestResponse.java
package fr.lewon.web.bot.entities.response;

import com.fasterxml.jackson.annotation.JsonProperty;

import fr.lewon.web.bot.entities.diverse.Cost;

public class ContinueQuestResponse extends Response {

	@JsonProperty("id_quest")
	private Long idQuest;

	@JsonProperty
	private Cost cost;

	public Long getIdQuest() {
		return idQuest;
	}

	public void setIdQuest(Long idQuest) {
		this.idQuest = idQuest;
	}

	public Cost getCost() {
		return cost;
	}

	public void setCost(Cost cost) {
		this.cost = cost;
	}

}

File: src/main/kotlin/fr/lewon/bot/hh/entities/response/ContinueQuestResponse.kt
package fr.lewon.bot.hh.entities.response

import com.fasterxml.jackson.annotation.JsonProperty
import fr.lewon.bot.hh.entities.diverse.Cost

class ContinueQuestResponse : Response() {
    @JsonProperty("id_quest")
    var idQuest: Long? = null
    @JsonProperty
    var cost: Cost? = null

}